### PR TITLE
[Blog] add 2.7.4 release blog

### PR DESCRIPTION
**Pull request by Anonymitaet**
Do not merge this PR since 2.7.4 is not announced officially and https://github.com/apache/pulsar/pull/13391 is not merged (that is the TBD link in this release blog).

Preview looks good:
image

### CODE REVIEW

File: site2/website/blog/2021-12-14-Apache-Pulsar-2-7-4.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
+---
+author: Technoboy-, Anonymitaet
+title: What’s New in Apache Pulsar 2.7.4
+---
+
+The Apache Pulsar community releases version 2.7.4! 32 contributors provided improvements and bug fixes that delivered 98 commits.
+
+Highlights of this release are as below:
+
+- `ManagerLedger` can be referenced correctly when `OpAddEntry` is recycled. [PR-12103](https://github.com/apache/pulsar/pull/12103)
```
Comment (codelipenghui) [inline site2/website/blog/2021-12-14-Apache-Pulsar-2-7-4.md:10]: We should highlight the log4j CVE fix.

Comment (Anonymitaet) [reply site2/website/blog/2021-12-14-Apache-Pulsar-2-7-4.md:12]: Good point. I've added, PTAL.

**Review by codelipenghui — APPROVED** [review]

**Review by Huanli-Meng — APPROVED** [review]
LGTM

File: site2/website/blog/2021-12-14-Apache-Pulsar-2-7-4.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+### Upgrade Log4j to 2.17.0 - [CVE-2021-45105](https://pulsar.apache.org/blog/2021/12/11/Log4j-CVE/). [PR-13392](https://github.com/apache/pulsar/pull/13392)
+
+- **Issue**
+
+    A serious vulnerability was reported regarding Log4j that can allow remote execution for attackers. The vulnerability issue is described and tracked under [CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228).
+
+- **Resolution**
+  
+  Patch your systems as soon as possible by setting the Java property to `-Dlog4j2.formatMsgNoLookups=true` and set the environment variable `LOG4J_FORMAT_MSG_NO_LOOKUPS=true`. Details see [Log4j2 Zero Day vulnerability (CVE-2021-44228)](https://pulsar.apache.org/blog/2021/12/11/Log4j-CVE/).
```
Comment (lhotari) [inline site2/website/blog/2021-12-14-Apache-Pulsar-2-7-4.md:28]: This is not necessary with Pulsar 2.7.4, please remove.

**Review by lhotari — CHANGES_REQUESTED** [review]

File: site2/website/blog/2021-12-14-Apache-Pulsar-2-7-4.md (hunk 2 above)
Comment (Anonymitaet) [reply site2/website/blog/2021-12-14-Apache-Pulsar-2-7-4.md:28]: @lhotari why not include this? 2.7.4 includes this important bug fix. Users need to know that, right?

Comment (lhotari) [reply site2/website/blog/2021-12-14-Apache-Pulsar-2-7-4.md:28]: The resolution part is not applicable for 2.7.4

Comment (Anonymitaet) [reply site2/website/blog/2021-12-14-Apache-Pulsar-2-7-4.md]: Oh, I see. I've updated, can you PTAL? @lhotari

Comment (lhotari) [reply site2/website/blog/2021-12-14-Apache-Pulsar-2-7-4.md]: Thanks, LGTM

**Review by lhotari — APPROVED** [review]

**Pull request merged**